webapp/app/model/form_data.py

```diff
@@ -29,6 +29,11 @@ class FamilienstandModel(BaseModel):
     familienstand_confirm_zusammenveranlagung: Optional[bool]
 
 
+class DisabilityModel(BaseModel):
+    person_a_has_disability: str
+    person_b_has_disability: Optional[str]
```

Comment:
Honestly, I think we can get rid of this and just reference the fields in the preconditions where they are required. Except, you want to introduce a complete model representing the disability logic?

Reply:
In `ShowPersonBPrecondition` there is a model that representing a part of the session data. Thought it is that not everything on the model needs to be validated and is only partially verified. How can I add just references of the fields in the preconditions?

Reply:
Got it thanks!